packages/node_modules/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -66,6 +66,7 @@ describe('plugin-device', () => {
 
           it('should call \'checkNetworkReachability()\'', () => {
             assert.called(device.checkNetworkReachability);
+            assert.isTrue(device.isReachabilityChecked);
```

Comment:
We try to limit unit test assertions to one per `it` so the assertion matches the description. I know that doesn't seem to be the case all over the sdk, but that is what we are aiming for. It would be better if you put this assertion in its own block, that way, if the unit test fails we know what specifically failed by the name of the block.

```js
it('should set isReachabilityChecked to true', () => {
  assert.isTrue(device.isReachabilityChecked);
});
```